src/component/admin/users/UsersList/UsersList.tsx

```diff
@@ -104,7 +103,7 @@ const UsersList = () => {
                     user={user}
                     openPwDialog={openPwDialog}
                     openDelDialog={openDelDialog}
-                    location={location}
+                    location={locationSettings}
```

Comment:
I think we should try to keep the props consistent aswell through components. Tracking the data flow gets trickier if the name of the property changes through the component tree.